src/h5web/vis-packs/core/shared/VisCanvas.module.css

```diff
@@ -13,6 +13,7 @@
 
 /* R3F's root element */
 .canvasWrapper {
+  composes: no-invert-canvas from global;
```

Comment:
R3F doesn't provide a way to add a class to the canvas, and composing a class from an element selector like `.canvasWrapper > canvas { composes: no-invert from global; }` is technically not possible.